packages/fxa-settings/src/pages/Signin/Signin/en.ftl

```diff
@@ -0,0 +1,17 @@
+# <span> is for visual effect
+signin-password-needed-header = Enter your password <span>for your { -product-firefox-account }</span>
+
+# $serviceLogo - "Continue to" is followed by an image of the logo for the service which the user is authenticating for. alt text for this image is the service name.
+# For languages structured like English, the phrase can read "to continue to"
+signin-subheader-with-logo = Continue to
+
+# $serviceName - the name of the service which the user authenticating for
+# For languages structured like English, the phrase can read "to continue to { $serviceName }"
+signin-subheader-without-logo = Continue to { $serviceName }
+signin = Sign in
+# <br /> is just spacing for visual display. <a> links to the Terms of Service and also to the Privacy Notice.
+firefox-and-pocket-tos = By proceeding, you agree to: <br /> { product-pocket }’s <a>Terms of Service</a> and <a>Privacy Notice</a><br />{ -brand-firefox}’s <a>Terms of Service</a> and <a>Privacy Notice</a>.
```

Comment:
Q: Should there be a period after the Pocket Privacy Notice link? Or should we remove the period after the Firefox Privacy Notice link? Feels inconsistent.

![](https://user-images.githubusercontent.com/11150372/213280308-7f3ddfdd-e76c-44b9-82c3-4ac3c0163329.png)

Reply:
That's a good question -- I think the implication is that it's a full sentence including both, but it doesn't quite scan that way does it?

Reply:
I reached out to Vesta to ask her about this! I think we're good to move ahead with what we have for now since this is what currently exists, but I'll ticket it if a response comes back later

Reply:
Just noting here that Jeff Pfaller commented in #fxa that there should not be a period here.
"There should be no period at the end of the 2nd one - this kind of feels like a bulleted list with text links, and we tend to not use end punctuation for either"

I'm updating `Signup` to follow this advice too :)